### ISSUE TITLE
example to align s223:Sensor  with brick:*_Sensor

### ISSUE THREAD

**Issue by VladimirAlexiev** [open223/docs.open223.info#43]
https://docs.open223.info/guides/brick-rec.html#using-brick-points-with-223 shows this:
```ttl
bldg:sensor a s223:Sensor ;
    s223:hasObservationLocation bldg:damper-out ;
    s223:hasPhysicalLocation bldg:damper ;
    s223:observes bldg:air-temp .

bldg:air-temp a s223:QuantifiableObservableProperty,
                brick:Supply_Air_Temperature_Sensor ;
    qudt:hasQuantityKind qudtqk:Temperature ;
    s223:hasAspect s223:Role-Supply ;
    qudt:hasUnit unit:DEG_C .
```
But why is the `s223:QuantifiableObservableProperty` instantiated as `brick:Supply_Air_Temperature_Sensor`?
I think that the `s223:Sensor` should be instantiated as `brick:Supply_Air_Temperature_Sensor`.

If so, then https://github.com/open223/docs.open223.info/pull/42 also needs to be adjusted.

**Comment by steveraysteveray** [comment]
I agree with @VladimirAlexiev, although I think he meant to say that bldg:sensor should be instantiated as brick:Supply_Air_Temperature_Sensor, not the class s223:Sensor.

**Comment by VladimirAlexiev** [comment]
By "the property" and "the sensor" I mean instances of these respective classes
